client/src/containers/Workspaces/Workspaces.js

```diff
@@ -1,69 +1,58 @@
-import React, { useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
 
-import Container from '../../components/Container/Container';
-import Columns from '../../components/Columns/Columns';
-import InputField from '../../components/InputField/InputField';
-import SidebarList from '../../components/SidebarList/SidebarList';
-import Spinner  from '../../components/Spinner/Spinner';
- 
+import Container from "../../components/Container/Container";
+import Columns from "../../components/Columns/Columns";
```

Comment:
These components now can be pulled from a single source.

Reply:
reverted this file